src/components/Progress/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC } from 'react';
+import Box from '../Box';
+import { Step } from './style';
+
+type PropsType = {
+    'data-testid'?: string;
+    current: number;
+    total: number;
+    paginateBy?: number;
```

Comment:
Maybe you could make this an enum like:
```suggestion
    paginateBy?: 3 | 5 | 7 | 9 | 11 | 13 | 15 | 17 | 19 | 21;
```

To prevent (console) errors. And also make sure people don't do absurdly long progress bolletjes. 21 is probably also retardedly long, so I guess `3 | 5 | 7 | 9 | 11` will be enough.